modules/text_server_adv/script_iterator.h

```diff
@@ -42,6 +42,8 @@
 #include <hb-icu.h>
 #include <hb.h>
 
+#define PAREN_STACK_DEPTH 128
```

Comment:
`static const int` should be preferred for static constants over `#define`: https://github.com/godotengine/godot/issues/44860

Reply:
Fixed/